decisions/0004-planning-of-developments.md

```diff
@@ -0,0 +1,102 @@
+# Planning and coordination of developments in Materials Project Software
+
+## Context and Problem Statement
+
+If developers would like to contribute an interface or workflow to the Materials Project software infrastructure, it is currently not possible to find out if other developers have already started with this development (except pull requests or issues have been made). 
+This could potentially lead to duplicated work. Could we encourage developers to at least raise an issue or similar before working on such an interface which is potentially of interest for many users and developers?
+
+One example for this are the LAMMPS workflows where multiple teams were interested in their development and only found out about each other by chance: https://github.com/materialsproject/atomate2/issues/173
+
+
+<!-- This is an optional element. Feel free to remove. -->
+## Decision Drivers
+
+* Collecting such planned developments could foster collaboration 
+* Collaboration could lead to faster availability of certain features
+* It would avoid duplication of work
+* It could strengthen the Materials Project software infrastructure and also attract new external developers and contributors
+* Developers would need to disclose a part of their project early on
+* We would need to encourage developers to communicate with us
+
+## Considered Options
+**Issues in individual software packages**
+ * Good because it is at least somehow documented
+ * Bad because developments might be in multiple packages and hard to discover
+ * Bad as it might distract from bugs or other relevant issues and add noise to the repo
+
+**Discussions in individual software packages**
+ * Good because it is documented
+ * Good because it be easier to maintain
+ * Bad because maintainers might overlook these discussions
+ * Bad because maybe hard to discover
+
+**Public MP org github project board**
+ * Good because developers can see quickly and with once glance what developments are on-going
+ * Good because you can link developments in multiple software packages
+ * Bad because you need to know github project boards
+ * Bad because maybe hard to discover
+ * External users that are not members of MP cannot use it
+ 
+**Forum entries**
+ * Good because it is easy to use
+ * Bad because the forum might be hard to navigate 
```

Comment:
Also because developer-oriented discussions might get lost among all the user support questions